### DISCUSSION TITLE
video disappears when the browser is resized

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2969] [General]
https://user-images.githubusercontent.com/63556979/228811190-d2aada5e-3b0d-415e-8587-ee086c8c5994.mp4

**Comment by Mister-Hope** [comment]
Can not reproduce, give me a reproduction.

This is why I ask you to create issues.

**Comment by tadmi** [comment]
@Mister-Hope test [this](https://theme-hope.vuejs.press/guide/markdown/components.html#youtube) like in a video upper

**Reply by Mister-Hope** [reply]
https://user-images.githubusercontent.com/33315834/229107584-76f6945f-8aca-4289-a57d-f65e5c7c831f.mp4

**Reply by Mister-Hope** [reply]
If you saw a black screen, download it to see the video, I am not sure why is it not working on browser.

**Reply by tadmi** [reply]
observed when changing the size of the browser (not inside the window)

https://user-images.githubusercontent.com/63556979/229141643-07c4951c-005c-4280-8964-d8df85ce45a0.mp4

**Reply by Mister-Hope** [reply]
Emm, thanks for the explanation, I think this is a bug coming from browser.

The resize event is not called at these moments, so the video is not being resized.
